pokete_classes/detail.py

```diff
@@ -12,7 +12,14 @@ class Informer:
 
     @staticmethod
     def add(poke, figure, _map, _x, _y, in_deck=True):
-        """Adds a Pokete to the deck"""
+        """Adds a Poketes info to the deck
```

Comment:
Not sure about that one, but I think there is an apostrophe missing:
```suggestion
        """Adds a Pokete's info to the deck
```